src/NuGet.Indexing/PerfEventTracker.cs

```diff
@@ -7,6 +7,7 @@
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
+using NuGetGallery;
```

Comment:
Repeating here: Indexing already depends on NuGetGallery.Core, and NuGetGallery namespace gives us access to the DisposableAction class used below.

---
In reply to: [78817894](https://github.com/NuGet/NuGet.Services.Metadata/pull/87#discussion_r78817894) [](ancestors = 78817894)